modules/neo4j/src/test/java/org/testcontainers/containers/Neo4jContainerTest.java

```diff
@@ -26,12 +27,16 @@
 
     // See org.testcontainers.utility.LicenseAcceptance#ACCEPTANCE_FILE_NAME
     private static final String ACCEPTANCE_FILE_LOCATION = "/container-license-acceptance.txt";
+    public static final DockerImageName NEO4J_IMAGE = Neo4jTestImages.NEO4J_TEST_IMAGE;
 
     @Test
     public void shouldDisableAuthentication() {
 
         try (
-            Neo4jContainer<?> neo4jContainer = new Neo4jContainer<>(Neo4jTestImages.NEO4J_TEST_IMAGE).withoutAuthentication()
+            // withoutAuthentication {
+            Neo4jContainer<?> neo4jContainer = new Neo4jContainer<>(NEO4J_IMAGE)
```

Comment:
I think we should consider having all code includes with a String literal Docker image name, rather than a constant so that the code snippets in the docs are copy-paste ready. I get that this can add more overhead in maintaining the tests, so WDYT?

Right now, it is slightly inconsistent, check it in the rendered form here:
https://deploy-preview-5120--testcontainers.netlify.app/modules/databases/neo4j/

Reply:
I think I'd agree with @kiview on this; other bits of the test use a string rather than a constant.

I think I'd rather have copy-pastable code samples and have the problem of needing to update a bunch of identical strings. Having a constant makes the task of updating a version a little easier - but I think that task is always going to be overshadowed much more by the inconvenience of just remembering to make the change and raise a PR.

FWIW I'm sure we're very inconsistent in this advice elsewhere in our docs/code, for which I take the blame 😬 😂

Reply:
I added the concrete versions now for even easier copying ;)
The inconsistency will still be present because one functionality is only available when using an older major version.

Reply:
I think, in this case, the inconsistency is even nice and explicit and also explained in the associated paragraph. 
LGTM 👍 ❤️